style.css

```diff
@@ -0,0 +1,282 @@
+html {
+  background-color: #f4f5f6;
+}
+
+* {
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+body,
+nav {
+  max-width: 1310px;
+  min-width: 320px;
+  margin: auto;
+}
+
+.body {
+  background-color: #f4f5f6;
+}
+
+.title-global {
+  font-size: 16px;
+  line-height: 16px;
+  font-weight: bold;
+  text-transform: uppercase;
+  padding-bottom: 11px;
+  color: rgba(3, 3, 3, 0.4);
+}
+
+.bg-global-red {
+  background-color: #f72210;
+}
+
+.bg-global-blue {
+  background-color: #3581f4;
+}
+
+.color-global-blue {
+  color: #3581f4;
+}
+
+.color-global-red {
+  color: #f72210;
+}
+
+.custom-logo {
+  width: 25%;
```

Comment:
There is a Bootstrap class for `width: 25%;` property. It is `w-25`. By removing this line and using the Bootstrap counterpart, you will further decrease your CSS code. Be sure to do that.